### PR TITLE
feat(pkger): add pkg new operation to cli to create packages

### LINKED ISSUE #15699: API to generate manifest shell
For this sprint, just generate a shell over the API.

Next sprint, we will use this API to specify which resources should belong in the generated manifest.

### PR DESCRIPTION
**Pull request by jsteenb2**
Closes #15699 

This extends the influx cli with the ability to create packages.

- [x] [Well-formatted commit messages](https://www.conventionalcommits.org/en/v1.0.0-beta.3/)
- [x] Rebased/mergeable
- [ ] Tests pass

### CODE REVIEW

File: cmd/influx/pkg.go [hunk 1]
```diff
@@ -91,6 +103,83 @@ func pkgApply(orgID, path *string, hasColor, hasTableBorders *bool) func(*cobra.
...
 
+func pkgCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Create a reusable pkg to create resources in a declarative manner",
+	}
+
+	wd, err := os.Getwd()
+	if err != nil {
+		panic(err)
```
Comment (jsteenb2) [inline cmd/influx/pkg.go]: intentionally panicing here, it should never reach here, and if it does, we have bigger problems 🤷‍♂

File: cmd/influx/pkg.go [hunk 2]
```diff
@@ -91,6 +103,83 @@ func pkgApply(orgID, path *string, hasColor, hasTableBorders *bool) func(*cobra.
 	}
 }
 
+func pkgCreateCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "new",
+		Short: "Create a reusable pkg to create resources in a declarative manner",
+	}
+
+	wd, err := os.Getwd()
```
Comment (kelwang) [inline cmd/influx/pkg.go]: to make it nicer, i will also add an output flag, so user to put it to anywhere they want

Comment (jsteenb2) [reply cmd/influx/pkg.go]: totally agree, I added the following to handle that, the wd is purely for a default val.
https://github.com/influxdata/influxdb/pull/15937/files#diff-0ea570dc74aae6eb4768e34c0d584960R118

Comment (kelwang) [reply cmd/influx/pkg.go]: looks like you already have a flag

**Pull request merged**